dev/drivers/scripts/plots/wafs/jevs_wafs_atmos_plots.sh

```diff
@@ -40,25 +42,23 @@ export COMPONENT=wafs
 export RUN=atmos
 export VERIF_CASE=grid2grid
 
-export cyc=${cyc:-00}
-export cycle=t${cyc}z
-setpdy.sh
-. ./PDY
-export VDATE=${VDATE:-$PDYm1}
-
-export COMIN=${COMIN:-/lfs/h2/emc/vpppg/noscrub/$USER/evs/$evs_ver}
-export COMOUT=/lfs/h2/emc/ptmp/$USER/evs/$evs_ver/${STEP}/${COMPONENT}/${RUN}.${VDATE}
```

Comment:
@YaliMao-NOAA Please bring back this deleted COMOUT path and delete export COMROOT on line 47. 

export COMOUT=/lfs/h2/emc/ptmp/$USER/evs/$evs_ver/${STEP}/${COMPONENT}/${RUN}.${VDATE}